target/linux/ramips/image/mt7621.mk

```diff
@@ -140,6 +140,38 @@ define Device/alfa-network_quad-e4g
 endef
 TARGET_DEVICES += alfa-network_quad-e4g
 
+define Device/ampedwireless_ally-r1900k
```

Comment:
Please create a common definition Device/ampedwireless_ally to inherit the others from

Reply:
good idea, I missed that... thanks.